### DISCUSSION TITLE
[@types/figlet] Missing preloadFonts

### DISCUSSION THREAD

**Discussion by jcubic** [DefinitelyTyped/DefinitelyTyped#70763] [Issues with a @types package] [answered]
It's in the README:

https://www.npmjs.com/package/figlet

This is how you import fonts and allow using sync API.

The type should look like this:

```typescript
function preloadFonts(fonts: Fonts[], ready: () => void): void;
```

I can create  a PR.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
It looks like this was added in #70794
